frontend/app/src/store/blockchain/chains.ts

```diff
@@ -0,0 +1,69 @@
+import { EvmChain, EvmTokenKind } from '@rotki/common/lib/data';
+import { type Nullable } from '@rotki/common';
+import { type ActionDataEntry } from '@/store/types';
+
+export const evmChainsData: ActionDataEntry[] = [
```

Comment:
since this is not directly related to store maybe we should move that to a different directory. In general we should remove any types/utils etc that are unrelated to the stores from the store directory to make it easier to enable auto-import for stores